src/TuDa.CIMS.Web/Components/SubmitPopup.razor

```diff
@@ -0,0 +1,54 @@
+@* Submit Popup *@
+<MudDialog>
+    <TitleContent></TitleContent>
+    <DialogContent>
+        <MudContainer>
+            <MudGrid Class="justify-space-between" Style="max-width: 400px;">
+                <MudItem>
+                    <MudText Typo="Typo.h6">Arbeitsgruppe auswählen</MudText>
+                </MudItem>
+                <MudItem Class="d-flex align-items-center" Style="margin-bottom: 300px;">
+                    <MudAutocomplete T="string" Label="Professor" SearchFunc="@Search">
+                        <ItemTemplate>
+                            <MudItem Class="d-flex align-items-center">
+                                <MudButton>
+                                    <MudIcon Icon="@Icons.Material.Outlined.Info" Class="mr-2"></MudIcon>
+                                </MudButton>
+                                <MudText>@context</MudText>
+                            </MudItem>
+                        </ItemTemplate>
+                    </MudAutocomplete>
+                </MudItem>
```

Comment:
But then I'm not sure if this works here because `WorkingGroup` of type `WorkingGroup` would be needed for the method `Submit()` in the cs file for the MudDialog and in the cs file for the separate component for the content.